rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1033,6 +1038,31 @@ public long getFeePerKb(Object[] args)
         return bridgeSupport.getFeePerKb().getValue();
     }
 
+    public long getLockingCap(Object[] args) {
+        logger.trace("getLockingCap");
+
+        Coin lockingCap = bridgeSupport.getLockingCap();
+
+        // Parse value to BTC
+        return lockingCap.divide(Coin.COIN.getValue()).getValue();
+    }
+
+    public boolean increaseLockingCap(Object[] args) {
```

Comment:
Method should be increaseLockingCapInBTC()

Reply:
changed it to use satoshis as it was originally intended